scrapers/base_scraper.py

```
@@ -699,12 +696,12 @@ async def get_torrent_data(
     async def process_stream(
         self,
         stream_data: Dict[str, Any],
-        metadata: MediaFusionMetaData,
+        metadata: MetadataData,
         catalog_type: str,
         processed_info_hashes: set[str],
         season: int = None,
```

Ruff (0.14.10)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)